Resources/views/Collector/db.html.twig

```diff
@@ -411,3 +434,238 @@
 
     //]]></script>
 {% endblock %}
+
+{% block queries_v2 %}
```

Comment:
Wouldn't it be possible to avoiding duplicating the whole panel ? Having the whole panel duplicated to show the info makes the maintenance much harder, and makes it very likely to forget updating both of them in the future.

Note that making some changes to the way the panel looks like in older versions is OK, as long as it looks good.
for instance, if you can get rid of the query shrinking in the old panel too while keeping things looking good, it is fine with me (the shrinking logic is insane anyway).
Looking at the refactoring in both Symfony 2.7 and 2.8 may make the work a bit harder for you, but it will make the future maintenance a lot easier for me (and it may help documenting the necessary changes for the migration process as other bundles would also benefit from sharing as much markup as possible).